### DISCUSSION TITLE
Does devlake retrieve previous GitHub workflow attempts when scraping for data?

### DISCUSSION THREAD

**Discussion by Brend-Smits** [apache/devlake#7667] [Q&A]
Hi all,

I wonder if Devlake retrieves previous GitHub workflow run attempts or if it just retrieves the latest one at all times?
In GitHub you can 'rerun' a workflow run, which will retrigger the same workflow under the same conditions.

**Comment by klesh** [comment]
I don't have a direct answer to that, but it depends on what the API returns. Would you like to test it out and share the knowledge?
